### DISCUSSION TITLE
Customize notification area icons not working?

### DISCUSSION THREAD

**Discussion by jafin** [valinet/ExplorerPatcher#770] [Q&A]
When I try to change the state of an icon using the `"Customize notification area icons"` none of the values will apply after clicking OK.
I am attempting to change from "Only show notifications" to `"Show icon and notifications"`

The `Always show all icons` does work.

## Environment
- Windows 11 220000.438
- Patcher 22000.469.41.16
- Taskbar style: Windows 10

Also, on occasions going into this screen `Customize notification area icons` the explorer process crashes/restarts.

**Comment by leon-Glitchking** [comment]
i am getting this same issue

Windows 11 Pro
Version	21H2
OS build	22000.434

Patcher 22000.469.41.16
Taskbar style: Windows 10

**Comment by magnumchode** [comment]
I'm also having this same issue where "always show" does nothing.

Win 11 Pro 21H2
OS Build 22000.376
Patcher 22000.469.42.3
Taskbar Style: Win10

**Comment by magnumchode** [comment]
I wanted to add, you can drag the icons from the expanded icon area down into the tray to get them to display, to work around this issue.  Not sure if this was an old Win10 feature, but it's actually a lot easier anyway.
